string-manipulation-1/src/is-vowel.js

```diff
@@ -0,0 +1,27 @@
+/* exported isVowel */
+function isVowel(char) {
+  switch (char) {
+    case 'a':
+      return true;
+    case 'A':
+      return true;
+    case 'e':
+      return true;
+    case 'E':
+      return true;
+    case 'i':
+      return true;
+    case 'I':
+      return true;
+    case 'O':
+      return true;
+    case 'o':
+      return true;
+    case 'U':
+      return true;
+    case 'u':
+      return true;
+    default:
+      return false;
+  }
+}
```

Comment:
If you convert `char` to lowercase, then you don't need to worry about checking both lowercase and uppercase
```
function isVowel(character) {
  switch (character.toLowerCase()) {
    case 'a':
    case 'e':
    case 'i':
    case 'o':
    case 'u':
      return true;
    default:
      return false;
  }
}
```